packages/data-stores/src/site/test/selectors.ts

```diff
@@ -268,7 +268,7 @@ describe( 'getSiteOptions', () => {
 		launch_status: '',
 		jetpack: false,
 		is_fse_eligible: false,
-		is_fse_active: false,
+		logo: { id: 'logoId', sizes: [ 'small' ], url: 'logoURL' },
```

Comment:
What's this about?

Reply:
This was added to clear the typescript issue I was seeing. I modified the object, so I decided to add the `logo` property to fix the issue as well. It was initially added to the SiteDetails interface, but it was not added here.

Reply:
Ah, thanks for resolving the error!